src/main/webapp/script.js

```diff
@@ -0,0 +1,20 @@
+function passQuery() {
+  const query = new FormData(document.getElementById('query-form'));
+  const personType = query.get('person-type');
+  const location = query.get('location');
+
+  fetch('/query?person-type=' + personType + '&location=' + location)
+    .then((response) => {
+      if (response.ok) {
+        response.json().then((jsonResponse) => JSON.parse(jsonResponse))
+        .then((data) => {
+          // data is a 2D array, where the first row is a header row and all
+          // subsequent rows are one piece of data (e.g. for a state or county)
+          displayVisualization(data);
```

Comment:
since you dont have this function now, you could either do a console.log(data).

Reply:
I did have a log originally. But then I saw that Olivia named her function displayVisualization on her branch, so I changed it to this so that they can merge nicely.